apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ProposalOverview/ProposalOverview.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ButtonStyle,
+  FormGroup,
+  Icon,
+  IconNames,
+  Input,
+  InputSize,
+} from '@sovryn/ui';
+
+import { translations } from '../../../../../../../locales/i18n';
+import {
+  ProposalOverviewData,
+  ProposalType,
+} from '../../NewProposalForm.types';
+
+export type ProposalOverviewProps = {
+  value: ProposalOverviewData;
+  onChange: (value: ProposalOverviewData) => void;
+  proposalType: ProposalType;
+  onBack: () => void;
+};
+
+export const ProposalOverview: FC<ProposalOverviewProps> = ({
+  value,
+  onChange,
+  proposalType,
+  onBack,
+}) => {
+  const [form, setForm] = useState(value);
+
+  const isSubmitDisabled = useMemo(
+    () =>
+      !form.title &&
+      !form.description &&
+      !form.discussionURL &&
+      !form.description,
+    [form.description, form.discussionURL, form.title],
+  );
+  return (
+    <div className="flex flex-col gap-4 relative pt-4">
+      <button className="absolute left-0 -top-2" onClick={onBack}>
+        <Icon size={14} icon={IconNames.ARROW_BACK} />
+      </button>
+      <FormGroup label={t(translations.bitocracyPage.proposalOverview.title)}>
+        <Input
+          value={form.title}
+          onChangeText={title => setForm(form => ({ ...form, title }))}
+          size={InputSize.large}
+        />
+      </FormGroup>
+
+      <FormGroup
+        label={t(translations.bitocracyPage.proposalOverview.discussionURL)}
+      >
+        <Input
```

Comment:
There's no validation on any input but according to the story, it's a part of it. Look at points 2b,c,d,e in the proposal and implement proper validation.

Also, @john-light will need your input on point 2e (if you're not sure about it, consult with @creed-victor).